pkg/settlement/swap/chequebook/factory.go

```diff
@@ -156,3 +156,12 @@ func (c *factory) ERC20Address(ctx context.Context) (common.Address, error) {
 	}
 	return erc20Address, nil
 }
+
+// DiscoverFactoryAddress returns the canonical factory for this chainID
+func DiscoverFactoryAddress(chainID int64) (common.Address, bool) {
+	if chainID == 5 {
+		// goerli
```

Comment:
Log here: connect to default factory for goerli network

Reply:
introduced a log